experimental/web/testing/test-runner.html

```diff
@@ -16,15 +16,26 @@
   <link rel="icon" href="./ghost.svg" type="image/svg+xml">
 
   <style>
-    .test-output {
+    .test-output-container {
       font-family: monospace;
-      white-space: pre;
-      overflow-x: scroll;
+      background-color: #111122;
+      color: #88EE33;
+      overflow-wrap: anywhere;
+    }
+
+    .test-output {
+      white-space: pre-wrap;
+      display: inline;
     }
 
     .stderr {
       color: red;
     }
+
+    .start-of-line {
+      user-select: none;
```

Comment:
This lets you select the output without including the leading characters (which are useful to separate line breaks from overflow vs line breaks from a new line of output): 
![image](https://user-images.githubusercontent.com/4010439/188009935-a8226ac1-34e8-4c2f-a9dc-14f78957e738.png)